src/components/Header/index.js

```diff
@@ -1,8 +1,7 @@
 import React, { Component } from "react";
 import { withRouter } from "react-router";
-import { withScroll } from "react-window-decorators";
 import classNames from "classnames";
-
+import WithScroll from "../WithScroll";
```

Comment:
if it's HOC write his name for small letter `import withScroll from "../WithScroll";`